ansible_base/authentication/social_auth.py

```diff
@@ -189,12 +192,57 @@ def validate(self, serializer, data):
         return data
 
 
+def extract_id_token_claims(backend, response):
+    try:
```

Comment:
In your code below you wrap this function in this if. Lets make this method edal with that case itself incase we ever wanted to call it from somewhere else:

```suggestion
   if 'id_token' not in response:
       return {}
    try:
```

Reply:
fixed